octopusdeploy3-single-vm-windows/azuredeploy.json

```diff
@@ -0,0 +1,355 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "West US",
+      "allowedValues": [
```

Comment:
Removed the default location. 

I built the allowed values for the location by ensuring that the Resource Providers and Resource Types were available using the following PowerShell:

```powershell
PS> $resourceProviders = @("Microsoft.Storage/storageAccounts", "Microsoft.Network/publicIPAddresses", "Microsoft.Network/virtualNetworks", "Microsoft.Network/networkInterfaces", "Microsoft.Compute/virtualMachines", "Microsoft.Sql/servers")
PS> $hash = @{}; Get-AzureLocation |? Name -in $resourceProviders |% { $_.Locations |% { $hash[$_] = $hash[$_] + 1 } }; $hash.Keys |% { New-Object PSObject -Property @{Location = $_; Count = $hash[$_] } } |? { $_.Count -eq $resourceProviders.Count } | Select Location | Sort -Property Location

Location
--------
Central US
East Asia
East US
East US 2
Japan East
Japan West
North Europe
South Central US
Southeast Asia
West Europe
West US
```